src/promiseCache.ts

```diff
@@ -17,15 +17,14 @@
 
 import { Cache, EphemeralCache } from '@alexsasharegan/simple-cache';
 
-
 export interface CacheParams {
   cacheCapacity: number;
   cacheTTL: number;
 }
 export class PromiseCache<K, V> {
   private readonly cache: Cache<string, Promise<V>>;
 
-  constructor({ cacheCapacity, cacheTTL }: CacheParams) {
+  constructor({ cacheCapacity = 100, cacheTTL = 60_000 }: CacheParams) {
```

Comment:
Why choose defaults?

Reply:
no reason other than it was useful in my testing. will remove.

Reply:
[1a7a0f0](https://github.com/ardriveapp/promise-cache/pull/2/commits/1a7a0f0b0dbb5cd3d4751ef72675c9de56f2dbd3)